docs/buildah-add.md

```diff
@@ -27,6 +27,16 @@ BUILDAH\_HISTORY environment variable. `export BUILDAH_HISTORY=true`
 
 Sets the user and group ownership of the destination content.
 
+**--contextdir**
+
+Build context directory. Specifying a context directory causes Buildah to
+chroot into that context directory. This means copying files pointed at
+by symbolic links outside of the chroot will fail.
```

Comment:
IDK, should we mention that this is used only in conjunction with --ignorefile?  Perhaps:
```
Used only in conjunction with the `--ignorefile` option.
```

Reply:
If you're after the not-copying-anything-from-outside-of-the-context-dir behavior, then specifying a context directory, even without specifying a .dockerignore file, makes sense.